types/scheduler/index.d.ts

```diff
@@ -20,7 +20,7 @@ export const unstable_NormalPriority = 3;
 export const unstable_IdlePriority = 5;
 export const unstable_LowPriority = 4;
 export function unstable_runWithPriority<T>(priorityLevel: number, eventHandler: () => T): T | undefined;
-export function unstable_scheduleCallback(callback: FrameCallbackType, deprecated_options?: { timeout: number}): CallbackNode;
+export function unstable_scheduleCallback(priorityLevel: number, callback: FrameCallbackType, options?: { delay: number}): CallbackNode;
```

Comment:
Timeout is still an option: https://github.com/facebook/react/blob/d346d92084316ec10714b9ac4bfb1ebcfdf05fb9/packages/scheduler/src/Scheduler.js#L308 and both of them are optional

Reply:
@eps1lon Oops, missed that one. The timeout back in now and both timeout and delay are optional.